__playwright-tests__/server.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const express = require('express');
+
+const app = express();
+const port = 3000;
+
+const htmlIntro = `<!doctype html><html>`;
+const htmlOutro = `</html>`;
+
+app.get('/', (req, res) => {
+  res.send(`${htmlIntro}<body>testing 1 2 3</body>${htmlOutro}`);
+});
+
+app.get('/toolong', (req, res) => {
+  res.send(
+    `${htmlIntro}<body><img id="cloudImg" src="/blahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahbblahblahblahblahblahblahblahblahblahblahb"></body>${htmlOutro}`
```

Comment:
Should the image `src` have an image file extension?

Reply:
I don't think it matters, because the resulting image (if you look at the endpoint for that long link) defers to `cloud.png`, which does have an image extension. And, because it's loading from a file, express will automatically send a `Content-Type` header.